src/test/browser/things-view/thing-test.ts

```diff
@@ -169,6 +169,9 @@ describe('Thing', () => {
     expect(stringId).toEqual(`string-${escapeHtmlForIdClass('spaced string')}`);
   });
 
+  /**
+  TODO: Fix or remove test case (https://github.com/WebThingsIO/gateway/issues/2906)
+
   it('should reset property value when setProperty is rejected', async () => {
```

Comment:
instead of commenting if it does not have compilation errors I would `it.skip` it and leave the todo comment.